diff --git a/node_modules/@railgun-community/waku-relayer-client-web/dist/models/constants.js b/node_modules/@railgun-community/waku-relayer-client-web/dist/models/constants.js
index c5b98d5..bb8cec1 100644
--- a/node_modules/@railgun-community/waku-relayer-client-web/dist/models/constants.js
+++ b/node_modules/@railgun-community/waku-relayer-client-web/dist/models/constants.js
@@ -1,16 +1,16 @@
-export const WAKU_RAILGUN_PUB_SUB_TOPIC = '/waku/2/railgun-relayer';
+export const WAKU_RAILGUN_PUB_SUB_TOPIC = "/waku/2/railgun-relayer";
 export const WAKU_RAILGUN_DEFAULT_PEERS = [
-    '/dns4/fleet.wecamefromapes.com/tcp/8000/wss/p2p/16Uiu2HAkyCqm8368Z1Y1pvrBcQQtb66Mjj7uvyAKGdwur3uCbqfb',
-    '/dns4/fleet.wecamefromapes.com/tcp/8001/wss/p2p/16Uiu2HAmTCCZFbQxDNF1TL9TKSPXtfF9QJtNWhCFE5M8UNzk7DQv',
-    '/dns4/fleet.wecamefromapes.com/tcp/8002/wss/p2p/16Uiu2HAm2g9z6PyWCAe6RXXQto6ykD4EM6YH67mmQYfBQtb6ZxFi',
-    '/dns4/fleet.wecamefromapes.com/tcp/8003/wss/p2p/16Uiu2HAmDii55GGfHZBz1XsJD6xmmC22baFyYQMaygv8a1yiuvEw',
-    '/dns4/fleet.wecamefromapes.com/tcp/8004/wss/p2p/16Uiu2HAmCEDPo7wFZUdFT3iE8ntAxTxRSZHKs6cEf9Rg44NwduNr',
-    '/dns4/fleet.wecamefromapes.com/tcp/8005/wss/p2p/16Uiu2HAkzkgDGEEah58RpR8ooFg3YbRhxJZ3pRZiZm3KGWreBfiw',
-    '/dns4/fleet.wecamefromapes.com/tcp/60000/p2p/16Uiu2HAkyCqm8368Z1Y1pvrBcQQtb66Mjj7uvyAKGdwur3uCbqfb',
-    '/dns4/fleet.wecamefromapes.com/tcp/60001/p2p/16Uiu2HAmTCCZFbQxDNF1TL9TKSPXtfF9QJtNWhCFE5M8UNzk7DQv',
-    '/dns4/fleet.wecamefromapes.com/tcp/60002/p2p/16Uiu2HAm2g9z6PyWCAe6RXXQto6ykD4EM6YH67mmQYfBQtb6ZxFi',
-    '/dns4/fleet.wecamefromapes.com/tcp/60003/p2p/16Uiu2HAmDii55GGfHZBz1XsJD6xmmC22baFyYQMaygv8a1yiuvEw',
-    '/dns4/fleet.wecamefromapes.com/tcp/60004/p2p/16Uiu2HAmCEDPo7wFZUdFT3iE8ntAxTxRSZHKs6cEf9Rg44NwduNr',
-    '/dns4/fleet.wecamefromapes.com/tcp/60005/p2p/16Uiu2HAkzkgDGEEah58RpR8ooFg3YbRhxJZ3pRZiZm3KGWreBfiw',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8000/wss/p2p/16Uiu2HAkyCqm8368Z1Y1pvrBcQQtb66Mjj7uvyAKGdwur3uCbqfb',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8001/wss/p2p/16Uiu2HAmTCCZFbQxDNF1TL9TKSPXtfF9QJtNWhCFE5M8UNzk7DQv',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8002/wss/p2p/16Uiu2HAm2g9z6PyWCAe6RXXQto6ykD4EM6YH67mmQYfBQtb6ZxFi',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8003/wss/p2p/16Uiu2HAmDii55GGfHZBz1XsJD6xmmC22baFyYQMaygv8a1yiuvEw',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8004/wss/p2p/16Uiu2HAmCEDPo7wFZUdFT3iE8ntAxTxRSZHKs6cEf9Rg44NwduNr',
+	// '/dns4/fleet.wecamefromapes.com/tcp/8005/wss/p2p/16Uiu2HAkzkgDGEEah58RpR8ooFg3YbRhxJZ3pRZiZm3KGWreBfiw',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60000/p2p/16Uiu2HAkyCqm8368Z1Y1pvrBcQQtb66Mjj7uvyAKGdwur3uCbqfb',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60001/p2p/16Uiu2HAmTCCZFbQxDNF1TL9TKSPXtfF9QJtNWhCFE5M8UNzk7DQv',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60002/p2p/16Uiu2HAm2g9z6PyWCAe6RXXQto6ykD4EM6YH67mmQYfBQtb6ZxFi',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60003/p2p/16Uiu2HAmDii55GGfHZBz1XsJD6xmmC22baFyYQMaygv8a1yiuvEw',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60004/p2p/16Uiu2HAmCEDPo7wFZUdFT3iE8ntAxTxRSZHKs6cEf9Rg44NwduNr',
+	// '/dns4/fleet.wecamefromapes.com/tcp/60005/p2p/16Uiu2HAkzkgDGEEah58RpR8ooFg3YbRhxJZ3pRZiZm3KGWreBfiw',
 ];
 //# sourceMappingURL=constants.js.map
